### ISSUE TITLE
immunity bug with starstorm still persists

### ISSUE THREAD

**Issue by lavaluck12** [nuxlar/UmbralMithrix#17]
I have been following the threads of the immunity bug. Through my testing I still can't get past the immunity on phase 3. The crystals don't spawn :(

**Comment by nuxlar** [comment]
yeah im sorry i ran into another starstorm incompat and fixed that then forgot to test the crystal stuff. It WILL be fixed

**Comment by jorblax** [comment]
Holy crap I'm glad I checked here, I was going crazy thinking that I'd missed something.
